### PR TITLE
Added access_ip_v6 support

### PR DESCRIPTION
**Pull request by jtopjian**
This commit populates access_ip_v6 by either the AccessIPv6 attribute
or by finding the first available IPv6 address.

This commit retains the original feature of setting the default ssh
connection to the IPv4 address unless one is not found. IPv6 access
can still be enabled by explicitly setting it in the resource paramters.

### CODE REVIEW

File: builtin/providers/openstack/resource_openstack_compute_instance_v2.go [hunk 1]
```diff
@@ -262,42 +262,79 @@ func resourceComputeInstanceV2Read(d *schema.ResourceData, meta interface{}) err
...
+					hostv4 = address["addr"].(string)
 					break
 				}
 			}
 		}
 	}
-	d.Set("access_ip_v4", host)
-	d.Set("host", host)
+	d.Set("access_ip_v4", hostv4)
+	d.Set("host", hostv4)
```
Comment (jtopjian) [inline builtin/providers/openstack/resource_openstack_compute_instance_v2.go]: What is being set/used here?

Comment (jrperritt) [inline builtin/providers/openstack/resource_openstack_compute_instance_v2.go]: `host` is used in the `ConnInfo`. It is for remote execution (via SSH for example).

Comment (jtopjian) [inline builtin/providers/openstack/resource_openstack_compute_instance_v2.go]: Ah, OK. Is this different than the host setting on line 299/335?

Comment (jrperritt) [inline builtin/providers/openstack/resource_openstack_compute_instance_v2.go]: No, it should be the same host as on 299.

Comment (jtopjian) [inline builtin/providers/openstack/resource_openstack_compute_instance_v2.go]: Sorry - I was vague. Are both necessary? Or is one a duplicate of the other? If they are both necessary, what makes them different?

From grepping the current terraform code, This is the only provider doing `d.Set("host")`. There are a few results outside of this provider for `SetConnInfo`

Comment (jrperritt) [inline builtin/providers/openstack/resource_openstack_compute_instance_v2.go]: Oh, perhaps not then. At one point in time (maybe last year), the other providers had set `host` in the compute instance files. I guess it's not needed anymore.

Comment (jtopjian) [inline builtin/providers/openstack/resource_openstack_compute_instance_v2.go]: OK, I'd be happy to submit another PR with it removed. Or unless you don't mind if it's done in this one?

Comment (jrperritt) [inline builtin/providers/openstack/resource_openstack_compute_instance_v2.go]: Yeah, doing it in this one is fine.

**Comment by jrperritt** [comment]
It might be easier to close this PR and create a new one on the updated `openstack-gophercloud-v1.0` branch than to rebase this one. Up to you though.

**Comment by jtopjian** [comment]
Well that was awesome. :smile:

**Comment by jrperritt** [comment]
Well that was easier than it seemed....  :+1:
